lib/acts_as_taggable_on_mongoid/tag_list.rb

```diff
@@ -154,16 +154,18 @@ def conditional_clean_rules
     # :reek:FeatureEnvy
     # :reek:DuplicateMethodCall
     def extract_and_apply_options!(args)
-      options = args.extract_options!
+      dup_args    = args.dup
```

Comment:
Layout/SpaceAroundOperators: Operator `=` should be surrounded by a single space.